react/features/chat/components/native/Chat.js

```diff
@@ -81,57 +73,8 @@ class Chat extends AbstractChat<Props> {
      * @param {string} message - The chat message to display.
      * @returns {void}
      */
-    _onSend([ message ]) {
-        this.props._onSendMessage(message.text);
-    }
-
-    _renderMessage: Object => React$Element<*>
-
-    /**
-     * Renders a single message.
-     *
-     * @param {Object} messageProps - The message props object to be rendered.
-     * @returns {React$Element<*>}
-     */
-    _renderMessage(messageProps) {
-        const { currentMessage } = messageProps;
-
-        return (
-            <ChatMessage message = { currentMessage } />
-        );
-    }
-
-    _transformMessage: (Object, number) => Object;
-
-    /**
-     * Transforms a Jitsi message object to a format that gifted-chat can
-     * handle.
-     *
-     * @param {Object} message - The chat message in our internal format.
-     * @param {number} index - The index of the message in the array.
-     * @returns {Object}
-     */
-    _transformMessage(message, index) {
-        const system = message.messageType === 'error';
-
-        return (
-            {
-                _id: index,
-                createdAt: new Date(message.timestamp),
-                messageType: message.messageType,
-                system,
-                text: system
-                    ? this.props.t('chat.error', {
-                        error: message.error,
-                        originalText: message.message
-                    })
-                    : message.message,
-                user: {
-                    _id: message.id,
-                    name: message.displayName
-                }
-            }
-        );
+    _onSend(message) {
```

Comment:
why not remove this one and pass the prop directly to the ChatInputBar?

Reply:
Hah, after all the refactors... Very true, thanks :)